packages/shared/README.md

```diff
@@ -2,4 +2,4 @@
 
 This is the shared folder.
 
-Any files you want to share in here across both frontend and backend MUST be have the .shared extension.
```

Comment:
@stemmlerjs, why is this `.shared` rule? Should we also then prefix `packages\shared\src\ensureAndLoadEnv.ts`, `packages\shared\src\logger.ts`, etc.?